helm/defectdojo/values.yaml

```diff
@@ -233,11 +233,11 @@ django:
       limits:
         cpu: 2000m
         memory: 512Mi
-    app_settings:
+    appSettings:
       processes: 2
       threads: 2
-      # max_fd: 102400 # Uncomment to set the maximum number of file descriptors. If not set will be detected by uwsgi
-    enable_debug: false  # this also requires DD_DEBUG to be set to True
+      # maxFd: 102400 # Uncomment to set the maximum number of file descriptors. If not set will be detected by uwsgi
+    enableDebug: false  # this also requires DD_DEBUG to be set to True
```

Comment:
One more spot to update in `_helpers.tpl` (these changes haven't landed in `master` yet): https://github.com/DefectDojo/django-DefectDojo/blob/f4d0ab08670d8e1c9fdf97caa7929573a41e2f66/helm/defectdojo/templates/_helpers.tpl#L172

Reply:
oh, nice catch. FIxed